RDVFSharp/Entities/Battlefield.cs

```diff
@@ -165,10 +165,7 @@ public void TurnUpKeep()
 
             TurnOrder[currentFighter].Regen();
 
-            if (CheckIfFightIsOver())
-            {
-                return;
-            }
+            CheckIfFightIsOver();
```

Comment:
Do you really want to continue executing the code after the fight is over? This will probably fuck things up